File: php/7kyu - B.php
<?php

//Link -> https://www.codewars.com/kata/5841f680c5c9b092950001ae

function generate_integers(int $m, int $n): array {
    $sequence = array();

    for($i = $m; $i <= $n; $i++){
        array_push($sequence, $i);
    }

    return $sequence;
}

?>

File: scala/7kyu - B.scala
// Link -> https://www.codewars.com/kata/5467e4d82edf8bbf40000155

object Hello extends App{

  def descendingOrder(num: Int): Int = {
    var numString = num.toString

    var charList = numString.toList
    charList = charList.toList
    charList = charList.sorted
    charList = charList.reverse

    var endNumber = charList.mkString("")

    return endNumber.toInt
  }
}
